server/api/views.py

```diff
@@ -10,198 +10,271 @@
 from rest_framework.views import APIView #APIView
 from rest_framework.response import Response #APIView
 from rest_framework.decorators import api_view
-from .models import Course
+# from .models import Course
 from .serializers import *
 # setting up environmental variables
 env = environ.Env()
 environ.Env.read_env()
 
-@api_view(['POST'])
-def postTest(request):
-    """ Test db connection by posting fake data into test table
-    """   
-    if request.method == 'POST':
-        serializer = TestSerializer(data=request.data)
-        if serializer.is_valid():
-            serializer.save()
-            return Response(serializer.data, status=status.HTTP_201_CREATED)
-        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST) 
-
-
-@api_view(['POST'])
-def postCoursePrereqInfo(request):
-    """ Post Course into course prereq string table
-    """   
-    if request.method == 'POST':
-        serializer = CoursePreqInfoSerializer(data=request.data)
-        if serializer.is_valid():
-            serializer.save()
-            return Response(serializer.data, status=status.HTTP_201_CREATED)
-        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST) 
-
 
+# POSTING Methods
+#############################################################################################
 @api_view(['POST'])
-def postCourse(request):
+def postCourse2023(request):
     """ Insert course object into Course table
     """
     if request.method == 'POST':
-        serializer = CourseSerializer(data=request.data)
-        if serializer.is_valid():
-            serializer.save()
-            return Response(serializer.data, status=status.HTTP_201_CREATED)
-        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
-@api_view(['POST'])
-def postPrerequisite(request):
-    """ Insert prerequisite entry into Prerequisite table 
```

Comment:
Can we comment out postPrerequisite and postDependency like the rest of the functions instead of deleting them?

Reply:
Yes, I deleted them and pasted them in the bottom - ie all the deleted ones are commented out on the bottom.